proposals/CSP.md

```diff
@@ -290,8 +156,11 @@ Where SRI-hash means applying sub-resource-integrity checks based on the hash of
 rejecting the operation if the hash does not match whitelisted hashes,
 and script-src means rejecting operations that are not allowed by the CSP
 policy's directives for the source of scripts, e.g. script-src restricting origins.
-Note that `unsafe-eval` effectively *implies* `wasm-unsafe-eval`.
-### Examples
+Note that `unsafe-eval` effectively *implies* `wasm-eval`.
+
+On the event of failure, then an `EvalError` should be thrown by the appropriate JavaScript operation.
```

Comment:
Above you talk about `WebAssembly.EvalError??`. It seems they should be aligned.

Reply:
That is a TBD to find the right name.

Reply:
Fair, though what I meant is that they should probably both be TBD or both be `EvalError`? (Thanks for making the other changes!)

Reply:
Ah, 55a48c33c5c4ae16ad8ed048f8b2ffae5b52b03f resolves this.